rules/azure/initial_access_consent_grant_attack_via_azure_registered_application.toml

```diff
@@ -0,0 +1,63 @@
+[metadata]
+creation_date = "2020/09/01"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a user grants permissions to an Azure-registered application or when an administrator grants tenant-wide
+permissions to an application. An adversary may create an Azure-registered application that requests access to data such
+as contact information, email, or documents. The attacker then tricks an end user into granting that application consent
+to access their data either through a phishing attack, or by injecting illicit code into a trusted website. After the
+illicit application has been granted consent, it has account-level access to data without the need for an organizational
+account. Normal remediation steps, like resetting passwords for breached accounts or requiring Multi-Factor
+Authentication (MFA) on accounts, are not effective against this type of attack, since these are third-party
+applications and are external to the organization.
```

Comment:
@brokensound77 - I feel like this field contains information that's helpful to an analyst about to triage an alert from this rule, but it's quite long. Do you know if we can add line breaks in for when the UI renders and displays a rule?

Reply:
![image](https://user-images.githubusercontent.com/16747370/91937877-7fc55d80-ecb8-11ea-8d8f-c09085a2f98f.png)
I created this rule with multiple line breaks that were stripped via the UI (which matches what is done in this repo). Perhaps some of it can be moved to `note`, which allows for the line breaks and may be able to take advantage of markdown

Reply:
Agree with moving some of the description to the `note` field where you can format with markdown.

I'd probably just keep the first two sentences for the description.

>Identifies when a user grants permissions to an Azure-registered application or when an administrator grants tenant-wide permissions to an application. An adversary may create an Azure-registered application that requests access to data such as contact information, email, or documents.

Reply:
Thanks for the suggestions. I moved the detailed info and investigation notes to the notes field. I formatted it as a bulleted list until we decide whether we want to include headings in the investigation field or not.

Please let me know what you think. Cheers

<img width="810" alt="image" src="https://user-images.githubusercontent.com/56409778/92158276-e3e73f00-ede8-11ea-94ae-29cc2c7cbe61.png">